assets/naf-process-panama.sql.j2

```diff
@@ -0,0 +1,14 @@
+SELECT
+  PARSE_TIMESTAMP("%y%m%d%k%M", CONCAT(DA,TI)) timestamp,
+  RC AS callsign,
+  NA AS shipname,
+  RN AS registry_number,
+  IR AS internal_id,
+  IF(XR=RC OR XR="N",NULL,XR) AS imo,
```

Comment:
@smpiano let's change this line  
` IF(XR=RC OR XR="N",NULL,XR) AS imo,`
with
`XR AS external_id,`

it will be more clear if more nations join and we want to reuse this.